app/depict/src/main/java/org/openscience/cdk/depict/Depiction.java

```diff
@@ -0,0 +1,322 @@
+/* Copyright (C) 2015  The Chemistry Development Kit (CDK) project
```

Comment:
No need - there will likely be some work related patches in future but this was all spare time.